課題③/main.js

```diff
@@ -77,13 +79,36 @@
     showAllTaks();
   });
   
-  // タスク削除する
+  // クリックしたらボタンの機能を実行する
   document.getElementById('list').addEventListener('click', e => {
-    if (e.target.value === '削除') {
-      const eventParent = e.target.parentNode;
-      const eventGrandParent = eventParent.parentNode;
-      const taskId = Number(eventGrandParent.firstChild.textContent);
-      tasks.splice(taskId,1);
+    // タスクのIDを取得する機能
+    const eventParent = e.target.parentNode;
+    const eventGrandParent = eventParent.parentNode;
+    const taskId = Number(eventGrandParent.firstChild.textContent);
+
+    // タスクを削除する機能
+    if (e.target.classList == 'delete') {
```

Comment:
厳密等価演算子 `===` を利用された方がよいと思いました。
ここ以降も `==` が使われている個所が複数あるので見直していただきたいです 🙏 

```suggestion
    if (e.target.classList === 'delete') {
```